soh/soh/Enhancements/randomizer/3drando/hint_list/hint_list_item.cpp

```diff
@@ -1361,7 +1361,18 @@ void StaticData::HintTable_Init_Item() {
                                                            // /*spanish*/un destructor de cerraduras final
                                                           CustomMessage("a final lockpick", /*german*/"ein finaler Dietrich", /*french*/"un crochet à porte final")});
                                                            // /*spanish*/una apertura portentosa final
-
+    hintTextTable[RHT_OVERWORLD_KEY] = HintText(CustomMessage("an Overworld Key", /*german*/"ein Überwelt-Schlüssel", /*french*/"une clé de l'Overworld"),
```

Comment:
I would prefer separate item names for each key, but I can do that in polish phase so I won't block over it.